packages/core/core/src/services/document-service/transform/relations/utils/i18n.ts

```diff
@@ -31,9 +30,8 @@ export const getRelationTargetLocale = (
   if (isSourceLocalized && isTargetLocalized) {
     // Check the targetLocale matches
     if (targetLocale !== opts.sourceLocale) {
-      throw new errors.ValidationError(
-        `Relation locale does not match the source locale ${JSON.stringify(relation)}`
-      );
+      // If not, we will look for the relation that matches the source locale
+      return opts.sourceLocale;
     }
   }
```

Comment:
This could be supplified to:
```ts
// Both source and target locales should match 
if (isSourceLocalized && isTargetLocalized) {
   return opts.sourceLocale;
}

```